internal/store/migrations.go

```diff
@@ -1,5 +1,5 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Hmm, guessing you didn't mean to move this?

Reply:
That's weird. I have a plugin that takes care of that for me. 🤔  Let me revert that.